package.json

```diff
@@ -60,6 +61,8 @@
     "semantic-release": "^21.0.7"
   },
   "dependencies": {
-    "smartypants": "^0.2.2"
+    "@types/marked": "^5.0.1",
```

Comment:
If we drop this package we will have to bump the minimum peer dep to `^6` and drop `^4 || ^5`.

Since version `^4 || ^5` doesn't ship their own types :)

Reply:
The user should install that if they are using v4 or v5 with typescript so we don't need to include it in this package

Reply:
Sounds fair

Reply:
Will update the PR